newm-server/src/main/kotlin/io/newm/server/features/user/repo/UserRepositoryImpl.kt

```diff
@@ -196,20 +204,26 @@ internal class UserRepositoryImpl(
             user.websiteUrl?.let { entity.websiteUrl = it.orNull()?.asValidUrl() }
             user.twitterUrl?.let { entity.twitterUrl = it.orNull()?.asValidUrl() }
             user.instagramUrl?.let { entity.instagramUrl = it.orNull()?.asValidUrl() }
-            user.spotifyProfile?.let {
-                entity.spotifyProfile = it.substringBefore("?").orNull()?.also { profile ->
-                    spotifyProfileUrlVerifier.verify(profile, entity.stageOrFullName)
+            try {
+                user.spotifyProfile?.let {
+                    entity.spotifyProfile = it.substringBefore("?").orNull()?.also { profile ->
+                        spotifyProfileUrlVerifier.verify(profile, entity.stageOrFullName)
+                    }
                 }
-            }
-            user.soundCloudProfile?.let {
-                entity.soundCloudProfile = it.substringBefore("?").orNull()?.also { profile ->
-                    soundCloudProfileUrlVerifier.verify(profile, entity.stageOrFullName)
+                user.soundCloudProfile?.let {
+                    entity.soundCloudProfile = it.substringBefore("?").orNull()?.also { profile ->
+                        soundCloudProfileUrlVerifier.verify(profile, entity.stageOrFullName)
+                    }
                 }
-            }
-            user.appleMusicProfile?.let {
-                entity.appleMusicProfile = it.substringBefore("?").orNull()?.also { profile ->
-                    appleMusicProfileUrlVerifier.verify(profile, entity.stageOrFullName)
+                user.appleMusicProfile?.let {
+                    entity.appleMusicProfile = it.substringBefore("?").orNull()?.also { profile ->
+                        appleMusicProfileUrlVerifier.verify(profile, entity.stageOrFullName)
+                    }
                 }
+            } catch (exception: OutletProfileUrlVerificationException) {
+                val message = exception.message!!
```

Comment:
Let's ensure our catch block can't ever throw. Instead do `?: "Unknown Error"` or something like that.

Reply:
The OutletProfileUrlVerificationException has only one constructor that requires that always requires the message argument. But I'll add that.